frontend/src/components/Dashboard/Opening/RecentOpeningsDataTable/testData.ts

```diff
@@ -0,0 +1,55 @@
+import { ITableHeader } from "../../../../types/TableHeader";
```

Comment:
Rename this file as `testData` implies that this is used only for test

Reply:
changed to headerData file, with less duplication.